examples/angular-cli/package.json

```diff
@@ -18,22 +18,22 @@
     "test:watch": "jest --watch"
   },
   "dependencies": {
-    "@angular/common": "^7.2.1",
-    "@angular/compiler": "^7.2.3",
-    "@angular/core": "^7.2.1",
-    "@angular/forms": "^7.2.1",
-    "@angular/platform-browser": "^7.2.1",
+    "@angular/common": "^7.2.6",
+    "@angular/compiler": "^7.2.6",
+    "@angular/core": "^7.2.6",
+    "@angular/forms": "^7.2.6",
+    "@angular/platform-browser": "^7.2.6",
     "@angular/platform-browser-dynamic": "^7.2.6",
     "@ngrx/store": "^7.2.0",
-    "core-js": "^2.6.2",
+    "core-js": "^2.6.5",
     "rxjs": "^6.3.3",
     "rxjs-compat": "^6.4.0",
     "zone.js": "^0.8.29"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "^0.13.1",
-    "@angular/cli": "^7.3.2",
-    "@angular/compiler-cli": "^7.2.1",
+    "@angular-devkit/build-angular": "^0.11.4",
```

Comment:
Why going down from `0.13.1` to `0.11.4` ?

Reply:
typo i think